src/Cli/func/Common/ProcessManager.cs

```diff
@@ -26,6 +26,11 @@ public IEnumerable<IProcessInfo> GetProcessesByName(string processName)
                 .Select(p => new ProcessInfo(p));
         }
 
+        public void KillMainProcess()
+        {
+            Process.GetCurrentProcess().Kill();
```

Comment:
should this use the overload accepting a boolean? I think if we call this method, we want to go for the whole process tree.

Same could be true for the KillChildProcesses method maybe?

Reply:
Reverted this change, turns out no we don't want to pass 'true' here as we end up with the process not exiting